chatbots-telegram-azure-functions/build.gradle.kts

```
@@ -0,0 +1,15 @@
+plugins {
+    id("io.micronaut.build.internal.chatbots-azure-functions")
+}
+dependencies {
+    implementation(mn.micronaut.serde.jackson)
+    implementation(project(":chatbots-lambda"))
+    api(project(":chatbots-telegram-core"))
+}
+
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+        substitute(module("io.micronaut:micronaut-jackson-databind"))
+            .using(module("io.micronaut.serde:micronaut-serde-jackson:1.3.0"))
```

```suggestion
            .using(module("io.micronaut.serde:micronaut-serde-jackson:${mn.versions.micronaut.serialization.get()}"))
```

This is used a couple of places, and in one https://github.com/micronaut-projects/micronaut-chatbots/blob/d939a3dac1d75eebe0377e1967139ffe24b7eb3e/buildSrc/src/main/groovy/io.micronaut.build.internal.chatbots-api.gradle#L15 it specifies a different version...

We should make the above change in all places we do this, just to make sure we stay consistent